configure.ac

```diff
@@ -78,7 +79,7 @@ PKG_CHECK_MODULES(ALL, [
     gio-unix-2.0
     gio-2.0 >= gio_minver
     pango >= pango_minver
-    gtk+-3.0 >= gtk_minver
+    gtk-layer-shell-0 >= gtk-layer-shell-0_minver
```

Comment:
Why gtk+-3.0 dependency is removed?

Reply:
That's probably a bad cut and paste, will bring it back now